src/guide/1/pubsub/javascript/snippets/load-sdk.test.yml

```diff
@@ -0,0 +1,11 @@
+---
+name: index#load
```

Comment:
We already give that prefix (`index`) to other snippet tests try something else such as `pub-sub#create`.
This review applies for snippet file name too.